source/EffectfulForcing/MFPSAndVariations/ContinuityProperties.lagda

```diff
@@ -0,0 +1,913 @@
+      where
+       Ⅰ = ap
+            (λ - → to-bool (embedding-𝟚-ℕ -) (embedding-𝟚-ℕ-gives-boolean -))
+            (embedding-𝟚-ℕ-1-implies-is-₁ (α n) p)
+       Ⅱ = embedding-𝟚-ℕ-1-implies-is-₁ (α n) p ⁻¹
+
+point-of-lemma : (α : Cantor)
+               → point-of (to-cantor₀ α) ∼ embedding-𝟚-ℕ ∘ α
+point-of-lemma α = λ _ → refl
+
+to-bool-lemma₁ : (α : Baire) (bv : is-boolean-point α) (i : ℕ)
+              → α i ＝ 0 → to-bool (α i) (bv i) ＝ ₀
+to-bool-lemma₁ α bv i p = ap (to-bool (α i)) †
+  where
+   † : bv i ＝ inl p
+   † = being-boolean-is-prop α i (bv i) (inl p)
+
+to-bool-lemma₂ : (α : Baire) (bv : is-boolean-point α) (i : ℕ)
+               → α i ＝ 1 → to-bool (α i) (bv i) ＝ ₁
+to-bool-lemma₂ α bv i p = ap (to-bool (α i)) †
+  where
+   † : bv i ＝ inr p
+   † = being-boolean-is-prop α i (bv i) (inr p)
+
+to-cantor₀-cancels-to-cantor : to-cantor₀ ∘ to-cantor ∼ id
+to-cantor₀-cancels-to-cantor (α , bv) = to-subtype-＝ being-boolean-point-is-prop †
+  where
+   ‡₁ : (i : ℕ) → α i ＝ 0 → embedding-C-B (to-cantor (α , bv)) i ＝ α i
+   ‡₁ i p = embedding-𝟚-ℕ (to-bool (α i) (bv i)) ＝⟨ Ⅰ ⟩
+            0                                    ＝⟨ p ⁻¹ ⟩
+            α i                                  ∎
+             where
+              Ⅰ = ap embedding-𝟚-ℕ (to-bool-lemma₁ α bv i p)
+
+   ‡₂ : (i : ℕ) → α i ＝ 1 → embedding-C-B (to-cantor (α , bv)) i ＝ α i
+   ‡₂ i p = embedding-C-B (to-cantor (α , bv)) i ＝⟨ refl ⟩
+            embedding-𝟚-ℕ (to-bool (α i) (bv i)) ＝⟨ Ⅰ ⟩
+            embedding-𝟚-ℕ ₁                      ＝⟨ refl ⟩
+            1                                    ＝⟨ p ⁻¹ ⟩
+            α i                                  ∎
+             where
+              Ⅰ = ap embedding-𝟚-ℕ (to-bool-lemma₂ α bv i p)
+
+   ‡ : embedding-C-B (to-cantor (α , bv)) ∼ α
+   ‡ i = embedding-C-B (to-cantor (α , bv)) i ＝⟨ refl ⟩
+         embedding-𝟚-ℕ (to-bool (α i) (bv i)) ＝⟨ Ⅰ    ⟩
+         α i                                  ∎
+          where
+           Ⅰ : embedding-𝟚-ℕ (to-bool (α i) (bv i)) ＝ α i
+           Ⅰ = cases (‡₁ i) (‡₂ i) (bv i)
+
+
+   † : embedding-C-B (to-cantor (α , bv)) ＝ α
+   † = dfunext fe ‡
+
+cantor-equiv-cantor₀ : Cantor ≃ Cantor₀
+cantor-equiv-cantor₀ = to-cantor₀ , (to-cantor , φ) , to-cantor , ψ
+ where
+  φ : to-cantor₀ ∘ to-cantor ∼ id
+  φ = to-cantor₀-cancels-to-cantor
+
+  ψ : to-cantor ∘ to-cantor₀ ∼ id
+  ψ = to-cantor-cancels-to-cantor₀
+
+\end{code}
+
+Now, we start working towards showing the equivalence of the alternative notion
+of uniform continuity with the original one. We define the following function
+`sequentialize` that flattens a binary tree into a list.
+
+\begin{code}
+
+sequentialize : {X : 𝓤₀  ̇} → BT X → List X
+sequentialize []      = []
+sequentialize (x ∷ φ) = x ∷ sequentialize (φ ₀) ++ sequentialize (φ ₁)
+
+\end{code}
+
+Recall the `maximum` operation that we used in the proof of
+`continuity₀-iff-continuity`. We now define an analogue of this operation for
```

Comment:
If you rename the above rename this too.

Reply:
Done.